pkg/scheduler/scheduler.go

```diff
@@ -109,12 +110,20 @@ func WithBindTimeoutSeconds(bindTimeoutSeconds int64) Option {
 	}
 }
 
+// WithPodMaxBackoffDurationSecond sets podMaxBackoffDurationSeconds for Scheduler, the default value is 10.
+func WithPodMaxBackoffDurationSecond(podMaxBackoffDurationSeconds int64) Option {
```

Comment:
This seems not  be called in cmd package ?

Reply:
addressed